aten/src/ATen/native/cpu/HistogramKernel.cpp

```diff
@@ -149,7 +149,7 @@ void histogramdd_cpu_contiguous(Tensor& hist, const TensorList& bin_edges,
                 const input_t elt = accessor_in[i][dim];
 
                 // Skips elements which fall outside the specified bins
-                if (elt < leftmost_edge[dim] || rightmost_edge[dim] < elt) {
+                if (!(elt >= leftmost_edge[dim] && elt <= rightmost_edge[dim])) {
```

Comment:
This seems OK but let's extend the above comment that this check skips elements that fall outside the bins AND NaNs